app/src/main/res/values-ru/strings.xml

```diff
@@ -491,24 +533,158 @@
+    <!-- An option to connect additional devices -->
+    <string name="sync_connect_device">Подключить другое устройство</string>
+    <!-- The dialog text shown when additional devices are not available -->
+    <string name="sync_connect_device_dialog">Чтобы отправить вкладку, войдите в Firefox хотя бы на ещё одном устройстве.</string>
+    <!-- Confirmation dialog button -->
+    <string name="sync_confirmation_button">Понятно</string>
+
     <!-- Notifications -->
     <!-- Text shown in snackbar when user deletes a collection -->
     <string name="snackbar_collection_deleted">Коллекция удалена</string>
 
+    <!-- Text shown in snackbar when user renames a collection -->
+    <string name="snackbar_collection_renamed">Коллекция переименована</string>
     <!-- Text shown in snackbar when user deletes a tab -->
     <string name="snackbar_tab_deleted">Вкладка удалена</string>
 
+    <!-- Text shown in snackbar when user deletes all tabs -->
+    <string name="snackbar_tabs_deleted">Вкладки удалены</string>
     <!-- Text for action to undo deleting a tab or collection shown in snackbar -->
     <string name="snackbar_deleted_undo">ОТМЕНИТЬ</string>
 
     <!-- QR code scanner prompt which appears after scanning a code, but before navigating to it
         First parameter is the name of the app, second parameter is the URL or text scanned-->
     <string name="qr_scanner_confirmation_dialog_message">Разрешить %1$s открыть %2$s</string>
+    <!-- QR code scanner prompt dialog positive option to allow navigation to scanned link -->
+    <string name="qr_scanner_dialog_positive">РАЗРЕШИТЬ</string>
+    <!-- QR code scanner prompt dialog positive option to deny navigation to scanned link -->
+    <string name="qr_scanner_dialog_negative">ЗАПРЕТИТЬ</string>
+    <!-- Tab collection deletion prompt dialog message. Placeholder will be replaced with the collection name -->
+    <string name="tab_collection_dialog_message">Вы уверены, что хотите удалить %1$s?</string>
+    <!-- Tab collection deletion prompt dialog option to delete the collection -->
+    <string name="tab_collection_dialog_positive">Удалить</string>
+    <!-- Tab collection deletion prompt dialog option to cancel deleting the collection -->
+    <string name="tab_collection_dialog_negative">Отмена</string>
     <!-- Text displayed in a notification when the user enters full screen mode -->
     <string name="full_screen_notification">Вход в полноэкранный режим</string>
 
     <!-- Message for copying the URL via long press on the toolbar -->
     <string name="url_copied">URL скопирован</string>
 
 
-    </resources>
+    <!-- Sample text for accessiblity font size -->
+    <string name="accessibility_text_size_sample_text">The quick brown fox jumps over the lazy dog. Съешь же ещё этих мягких французских булок, да выпей чаю. The quick brown fox jumps over the lazy dog. Съешь же ещё этих мягких французских булок, да выпей чаю.</string>
```

Comment:
Is this correct?

Reply:
Yes, they're just trying to cover both the latin and the cyrrillic alphabet. There's a Chinese localization doing a similar thing.